diff --git a/swiftmailer.module b/swiftmailer.module
index 2fb3265..241100a 100644
--- a/swiftmailer.module
+++ b/swiftmailer.module
@@ -39,6 +39,19 @@ define('SWIFTMAILER_VARIABLE_SENDMAIL_MODE_DEFAULT', 'bs');
 define('SWIFTMAILER_VARIABLE_CHARACTER_SET_DEFAULT', 'UTF-8');
 
 /**
+ * Implements hook_permission().
+ */
+function swiftmailer_permission() {
+
+  return array(
+    'swiftmailer_administer' => array(
+      'title' => t('Administer Swift Mailer'),
+    ),
+  );
+
+}
+
+/**
  * Implements hook_menu().
  */
 function swiftmailer_menu() {
